### ISSUE TITLE
When build x86 mode with multi_span

### ISSUE THREAD

**Issue by ajtruckle** [microsoft/GSL#1004]
> 4>D:\My Libraries\vcpkg\installed\x86-windows\include\gsl\multi_span(86,74): warning C4003: not enough arguments for function-like macro invocation 'max'

Line 86 in `multi_span`:

        static const SizeType max_value = std::numeric_limits<SizeType>::max();

**Comment by beinhaerter** [comment]
That looks like you got the `max` macro from `Windows.h`.  Please check [this StackOverflow question](https://stackoverflow.com/questions/11544073/how-do-i-deal-with-the-max-macro-in-windows-h-colliding-with-max-in-std). The solution is to `#define NOMINMAX` before `#include <Windows.h>`.

**Comment by ajtruckle** [comment]
I closed this too early. I still get a warning even with that defined in my stdafx. But I am not going to worry for now. Levae it has it is for the moment.

**Comment by ajtruckle** [comment]
Yes, I downloaded the release. Why would the latest branch not be released over a year later? I’ll download you latest code then. vcpkg gave me headaches, ok for 32 and not for 64. And it had all those ports when all I want is gsl. I like things simple. Nuget is simple. And I never install pre-release as they are not classed stable.

Surely this lib should pushed and released after over a year? 👍😀

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by ajtruckle** [comment]
I have downloaded the latest code and compiled with that. No more warnings.
